requirements-dash.txt

```diff
@@ -0,0 +1,4 @@
+-e git+git://github.com/plotly/dash.git@master#egg=dash
+-e git+git://github.com/plotly/dash-core-components@master#egg=dash-core-components
+-e git+git://github.com/plotly/dash-html-components@master#egg=dash-html-components
+-e git+git://github.com/plotly/dash-table@master#egg=dash-table
```

Comment:
So when would this actually get used? I guess if someone manually runs `pip install -r dev-requirements.txt` because they're editing renderer but not `dash`? If that's all we might not want to encourage this, instead directing people to a more complete dev setup.

Reply:
Maybe I remove this altogether?